spec/main-process/atom-application.test.js

```diff
@@ -749,8 +749,8 @@ describe('AtomApplication', function() {
       assert.isFalse(w._locations[0].isFile);
     });
 
-    it('truncates trailing whitespace and colons', async function() {
```

Comment:
Doesn't this affect Windows? File names with a colon in their name can't be written to the disk.

Reply:
It does affect windows. And you are right that filenames with colons cannot be written to disk on windows. We could do two things
- Add a condition check on `platform` and strip the `:` based on the platform.
- Not fix this bug

Adding a condition check on the code means we also have to do that in our tests, otherwise we will end up with failing tests on windows.

Reply:
Yes you should add a condition to do this only on Unix based systems.